sphinx/domains/python.py

```diff
@@ -62,6 +64,51 @@
 }
 
 
+def _parse_arglist(arglist: str) -> addnodes.desc_parameterlist:
+    """Parse a list of arguments using python parser"""
+    params = addnodes.desc_parameterlist(arglist)
+    sig = signature_from_str('(%s)' % arglist)
+    last_kind = None
+    for param in sig.parameters.values():
+        if param.kind != param.POSITIONAL_ONLY and last_kind == param.POSITIONAL_ONLY:
+            # PEP-570: Separator for Positional Only Parameter: /
+            params += nodes.inline('', '/', classes=['operator'])
```

Comment:
To go forward this, we need to discuss more about how to name CSS classes (refs: #4289)...